proposals/simd/SIMD.md

```diff
@@ -304,6 +330,21 @@ def S.shuffle(a, b, s):
     return result
 ```
 
+#### Variable permutation rule
+* `v8x16.permute_dyn(a: v128, s: v128) -> v128`
+
+Returns a new vector with lanes selected from the lanes of the first input vector
+`a` and specified in the second input vector `s`. The indices from `s` are first
+fit into the range `[0, 15]` via a modulo.
+
+```python
+def S.permute_dyn(a, s):
+    result = S.New()
+    for i in range(S.Lanes):
+      result[i] = a[s[i] % S.Lanes]
+    return result
+```
+
```

Comment:
Sorry for the churn here, but would it be possible to split these into their own separate pull requests so that they can be reviewed separately? There is consensus for the dynamic shuffles, but  the immediate packing, and additional shuffles below would benefit from more discussion.

Reply:
@dtig, see #68, #69, and #70.

Reply:
I see the issues, and exposing dynamic shuffles seems like the right way to move forward - I was suggesting splitting that out to it's own PR to review/merge instead of all three together.

Reply:
Oh gotcha, I had missed the PR/issue distinction :+1: